File: dotnet-app/PPPK_Projekt/Models/Vozilo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PPPK_Projekt.Models
{
    public class Vozilo
    {
        public int IDVozilo { get; set; }
        public string Tip { get; set; }
        public string Marka { get; set; }
        public int GodinaProizvodnje { get; set; }
        public int PrijedeniKilometri { get; set; }
        public bool JeSlobodno { get; set; }

        public string TipMarka => ToString();
        public override string ToString() => Tip + " - " + Marka;

        public Vozilo()
        {
        }

        public Vozilo(int iDVozilo, string tip, string marka, int godinaProizvodnje, int prijedeniKilometri, bool jeSlobodno)
        {
            IDVozilo = iDVozilo;
            Tip = tip;
            Marka = marka;
            GodinaProizvodnje = godinaProizvodnje;
            PrijedeniKilometri = prijedeniKilometri;
            JeSlobodno = jeSlobodno;
        }

        public Vozilo(string tip, string marka, int godinaProizvodnje, int prijedeniKilometri, bool jeSlobodno)
        {
            Tip = tip;
            Marka = marka;
            GodinaProizvodnje = godinaProizvodnje;
            PrijedeniKilometri = prijedeniKilometri;
            JeSlobodno = jeSlobodno;
        }
    }
}

File: Java-I2/src/hr/algebra/model/Vozilo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hr.algebra.model;

/**
 *
 * @author Mihael
 */
public class Vozilo {
    private int IDVozilo;
    private String Tip;
    private String Marka;
    private int GodinaProizvodnje;
    private int PrijedeniKilometri;
    private Boolean JeSlobodno;

    public Vozilo(String Tip, String Marka, int GodinaProizvodnje, int PrijedeniKilometri, Boolean JeSlobodno) {
        this.Tip = Tip;
        this.Marka = Marka;
        this.GodinaProizvodnje = GodinaProizvodnje;
        this.PrijedeniKilometri = PrijedeniKilometri;
        this.JeSlobodno = JeSlobodno;
    }

    public Vozilo(int IDVozilo, String Tip, String Marka, int GodinaProizvodnje, int PrijedeniKilometri, Boolean JeSlobodno) {
        this(Tip, Marka, GodinaProizvodnje,PrijedeniKilometri,JeSlobodno);
        this.IDVozilo = IDVozilo;
    }

    public int getIDVozilo() {
        return IDVozilo;
    }

    public String getTip() {
        return Tip;
    }

    public String getMarka() {
        return Marka;
    }

    public int getGodinaProizvodnje() {
        return GodinaProizvodnje;
    }

    public int getPrijedeniKilometri() {
        return PrijedeniKilometri;
    }

    public Boolean getJeSlobodno() {
        return JeSlobodno;
    }
    
    @Override
    public String toString() {
        return String.format("Kolegij %d, %s, %s, %d, %d, %s", IDVozilo, Tip, Marka, GodinaProizvodnje, PrijedeniKilometri, JeSlobodno);
    }
}
